lib/vanagon/platform/windows.rb

```diff
@@ -289,6 +289,105 @@ def output_dir(target_repo = "")
         File.join("windows", target_repo, @architecture)
       end
 
+      # Generate correctly formatted wix elements that match the
+      # structure of the directory input
+      #
+      # @param services, Array of components services
+      # and optionally:
+      # @return [string] correctly formatted wix element string
+      def generate_wix_dirs(services)
+        directories = []
+        services.map { |svc| directories.push({ :path => svc.service_file, :id => "#{svc.id}BINDIR" }) }
```

Comment:
Thats correct Melissa - in this case, they all are in the same directory, but the structure allows for a service file to be placed outside of the main bin directory.
Having multiple ID's point to the same actual directory doesn't appear to be a problem for the MSI.